docs/source/daml/intro/5_Restrictions.rst

```diff
@@ -0,0 +1,60 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Restricting the Contract Model
+==============================
+
+Oftentimes, contract models will have constraints on the data stored or the allowed data transformations. In this section, you will learn about the two main mechanisms provided in DAML:
```

Comment:
```suggestion
Oftentimes, contracts will have constraints on the data stored or the allowed data transformations. In this section, you will learn about the two main mechanisms provided in DAML:
```
What is a "contract model"? You have defined "contract" and "template".

Reply:
Defined it now.